app/main/helpers/frameworks.py

```diff
@@ -154,6 +154,12 @@ def return_supplier_framework_info_if_on_framework_or_abort(data_api_client, fra
 
 
 def question_references(data, get_question):
+    """
+    Replace placeholders for question references with the number of the referenced question
+
+    e.g. "This is my question hint which references [[anotherQuestion]]" becomes
+    "This is my question hint which references 7"
+    """
```

Comment:
It would be nice to document the *weird bit* of this function, which is the `get_question` argument - expected to be a bound callable from what i can remember.

Reply:
Good point, will add